packages/table/src/quadrants/tableQuadrant.tsx

```diff
@@ -99,14 +105,23 @@ export interface ITableQuadrantProps extends IProps {
 }
 
 export class TableQuadrant extends AbstractComponent<ITableQuadrantProps, {}> {
+    // we want the user to explicitly pass a quadrantType. define defaultProps as a Partial to avoid
+    // declaring that and other required props here.
+    public static defaultProps: Partial<ITableQuadrantProps> & object = {
+        isRowHeaderShown: true,
```

Comment:
I would, but there's no `isColumnHeaderShown` prop on `Table`, so we'd just need `isRowHeaderShown` here I believe.